docs/glossary.md

```diff
@@ -0,0 +1,388 @@
+  is an EPICS "configuration" file
+  containing only record instances/implementations definitions.
+  This file is like a list of records,
+  which will details their associated types, names and fields.
+
+  :::{seealso}
+
+  - <https://docs.epics-controls.org/en/latest/appdevguide/databaseDefinition.html?highlight=database%20file#dbloadrecords>
+  - <https://docs.epics-controls.org/en/latest/guides/EPICS_Process_Database_Concepts.html>
+  - <https://docs.epics-controls.org/projects/how-tos/en/latest/applications/common-database-patterns.html>
+  - <https://docs.epics-controls.org/projects/how-tos/en/latest/getting-started/creating-ioc.html?highlight=db#creation-of-an-input-output-controller-ioc>
+
+  :::
+
+**`.template` file**
+  A Template file (`.template`) is just like a `.db` file,
+  but also including macro(s).
+
+  :::{seealso}
+
+  - <https://docs.epics-controls.org/en/latest/appdevguide/databaseDefinition.html?highlight=template#example-7>
+  - <https://docs.epics-controls.org/en/latest/specs/EPICSBuildFacility.html?highlight=template#templates>
+
+  :::
+
+**`.dbd` file**
+  A DataBase Definition file is an EPICS "configuration" file
+  containing any sort of definitions except for record instances/implementations
+  (like found in the `.db` and `.template` files),
+  because record instances/implementations are fundamentally different
+  from the other definitions.
+
+  A file containing record instances
+  should never contain any of the other definitions and vice-versa.
+
+  The definitions covered by a `.dbd` file include "Menus", "Record Types", "Devices",
+  "Drivers", "Registrars", "Variables", "Functions", "Breakpoint Tables",
+  "Record Instances"...
+
+  :::{seealso}
+
+  <https://docs.epics-controls.org/en/latest/appdevguide/databaseDefinition.html>
+
+  :::
+
+**`.substitutions` file**
+  A `.substitutions` file is an EPICS "configuration" file
+  that allows to load one or more `.template` files, multiple times.
+
+  The syntax of a `.substitutions` file is covered in
+  [this documentation](https://docs.epics-controls.org/en/latest/appdevguide/databaseDefinition.html?highlight=template#template-file-syntax)
+  and the format of a `.substitutions` file is covered in
+  [this documentation](https://docs.epics-controls.org/en/latest/appdevguide/databaseDefinition.html?highlight=template#template-file-formats).
+
+  :::{seealso}
+
+  - <https://docs.epics-controls.org/en/latest/appdevguide/databaseDefinition.html?highlight=template#dbloadtemplate>
+
+  :::
+
+**`.cmd` file**
+  A `.cmd` file is an EPICS executable file
+  that starts with a [Shebang](https://en.wikipedia.org/wiki/Shebang_(Unix))
+  instructing the program loader to run the associated IOC program,
+  passing the content of the `.cmd` file as the first argument.
+
+  This `.cmd` file is mostly used to run the IOC program
+  after loading all the associated EPICS "configuration" files
+  (e.g. `.dbd`, `.db`, `.template`, `.substitutions`, etc).
```

Comment:
I think for a new user, the second description is going to be better.

I've also replaced `after` by `and`, because the `.cmd` files also does this.

```suggestion
  This `.cmd` file is mostly used to run the IOC program
  and loading all the associated EPICS "configuration" files
  (e.g. `.dbd`, `.db`, `.template`, `.substitutions`, etc).

  A `.cmd` file is an EPICS executable file
  that starts with a [Shebang](https://en.wikipedia.org/wiki/Shebang_(Unix))
  instructing the program loader to run the associated IOC program,
  passing the content of the `.cmd` file as the first argument.
```